src/PoolManager.sol

```diff
@@ -26,6 +26,54 @@ import {Extsload} from "./Extsload.sol";
 
 /// @notice Holds the state for all pools
 
+
+//  4
+//   44
+//     444
+//       444                   4444
+//        4444            4444     4444
+//          d444          4444444    4444                           4
+//            4444        44444444     4444                         4
+//             44444       4444444       4444444444444444       444444
+//           4   44444     44444444       444444444444444444444    4444
+//            4    44444    4444444         4444444444444444444444  44444
+//             4     444444  4444444         44444444444444444444444 44  4
+//              44     44444   444444          444444444444444444444 4     4
+//               44      44444   44444           4444444444444444444 4 44
+//                44       4444     44             44444444444d444     444
```

Comment:
```suggestion
//                44       4444     44             444444444444444     444
```

Reply:
theres a bunch of `d`s are they purposeful?

Reply:
oo no i can s/d/4